src/SwedbankPay.Sdk.Tests/Json/CustomCurrencyCodeConverterTests.cs

```diff
@@ -1,46 +1,43 @@
-﻿using System;
-
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
-
-using SwedbankPay.Sdk.Tests.TestBuilders;
-
-using Xunit;
-
-namespace SwedbankPay.Sdk.Tests.Json
-{
-    public class CustomCurrencyCodeConverterTests
-    {
-        private readonly string currencyCode = "SEK";
-        
-        [Fact]
-        public void CanDeSerialize_CurrencyCode()
-        {
-            //ARRANGE
-
-            var jsonObject = new JObject { { "currency", this.currencyCode } };
-
-            //ACT
-            var result = JsonConvert.DeserializeObject<CurrencyCode>(jsonObject.ToString(), JsonSerialization.JsonSerialization.Settings);
-
-            //ASSERT
-            Assert.Equal(this.currencyCode, result.ToString());
-        }
-
-
-        [Fact]
-        public void CanSerialize_CurrencyCode()
-        {
-            //ARRANGE
-            var builder = new PaymentOrderRequestBuilder();
-            var paymentOrderRequest = builder.WithTestValues(Guid.NewGuid()).Build();
-
-            //ACT
-            var result = JsonConvert.SerializeObject(paymentOrderRequest.PaymentOrder, JsonSerialization.JsonSerialization.Settings);
-            var obj = JObject.Parse(result);
-            obj.TryGetValue("Currency", StringComparison.InvariantCultureIgnoreCase, out var code);
-            //ASSERT
-            Assert.Equal(this.currencyCode, code);
-        }
-    }
+﻿using SwedbankPay.Sdk.Common;
+using SwedbankPay.Sdk.Tests.TestBuilders;
+using System;
+using System.Text.Json;
+using Xunit;
+
+namespace SwedbankPay.Sdk.Tests.Json
+{
+    public class CustomCurrencyCodeConverterTests
+    {
+        private readonly string currencyCode = "SEK";
+
+        [Fact]
+        public void CanDeSerialize_CurrencyCode()
+        {
+            //ARRANGE
+
+            var jsonObject = $"{{\"currency\": \"{this.currencyCode}\" }}";
+
+            //ACT
+            var result = JsonSerializer.Deserialize<CurrencyCode>(jsonObject.ToString(), JsonSerialization.JsonSerialization.Settings);

```

Comment:
Codacy found an issue: [There's no need to call 'ToString()' on a string.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)